protobuf/ckksparams.proto

```diff
@@ -7,5 +7,5 @@ package hit.protobuf;
 message CKKSParams {
     required bytes ctx = 1;
     required bytes pubkey = 2;
-    required bool standardParams = 3;
+    optional bytes btp_params = 3;
```

Comment:
Do we need to worry about deserializing old values which have a different value stored in `3`?

Reply:
It could be a problem for people who have saved keys, but it's simple enough to delete old keys and regenerate them. It's the cost of using a reseach library :) Another potential concern is that a user can't tell whether a serialized object was created by SEAL or by Lattigo. They don't know which branch to use.

A more robust library would have a better way to handle this, but I think it's a low priority at this point.